README.md

```diff
@@ -496,57 +385,63 @@ mvn clean verify -Dcode-quality
 
 Then open target/site/jacoco/index.html in your browser to see code coverage.
 
-If you want to see more of the code quality outputs, you must run the full site 
-generation from [above](code-quality)
+### Logging
 
-#### Install
+Logging is handled by [logback](Logback Home).  By default, the file logback.xml in the \<configDir\> will be 
+used.  You can point to another file with the *--logbackConfig* argument.  Feel free to modify the file
+in target/config if you want different logging.  The logback.xml is configured to only log to the console 
+currently.
 
-The *install* task runs after *verify* and copies any included
-artifacts made in the *package* phase to your local Maven repo.  This
-repository is usually ~/.m2/repository unless you set to be somewhere
-else.  See the [settings.xml.sample](settings.xml.sample) file for
-details about moving that off.  It is useful, for example, when your
-home directory is slow.  You can move your Maven repo to a faster
-drive.
+### Classpath
 
-Run the *install* task with
+The classpath is setup in one of 2 ways.  
 
-```
-mvn clean install
-```
+* #### Development
+  * If the *emissary* script is a sibling to pom.xml, it is assumed you are running
+from the git checkout. If a file named .mvn-classpath exists, that file is used to setup the classpath. If that file
```

Comment:
This is incorrect now. 

"If the *emissary* script is a sibling to pom.xml, it is assumed you are running from the git checkout. A file named .mvn-classpath will be created using a Maven command then the Emissary class is launched."